frontend/src/components/LineageCardRow.tsx

```diff
@@ -27,14 +37,30 @@ export class LineageCardRow extends React.Component<LineageCardRowProps> {
           <p className='rowTitle'>{title}</p>
           <p className='rowDesc'>{description}</p>
         </footer>
-        {this.checkEdgeAfforances()}
+        {this.checkEdgeAffordances()}
       </div>
     );
   }
+
   private checkRadio(): JSX.Element {
     if (!this.props.hideRadio) {
-      return <div><input type='radio' className='form-radio' name='' value='' /></div>;
+      return <div><input type='radio' className='form-radio' name='' value='' onClick={this.handleClick} /></div>;
     }
     return <div className='noRadio' />;
   }
-}
\ No newline at end of file
+
+  private get artifact(): Artifact {
+    // TODO: Return this.props.artifact once available.
+    const mockArtifact = new Artifact();
+    const nameValue = new Value();
+    nameValue.setStringValue(this.props.title);
+    mockArtifact.getPropertiesMap().set(ArtifactProperties.NAME, nameValue);
+    return mockArtifact
+  }
+
+  private handleClick() {
+    if (this.props.setLineageViewTarget) {
```

Comment:
Invert to decrease code depth
```suggestion
    if (!this.props.setLineageViewTarget) return;
```

Reply:
Done